website/static/schemas/next/context/timeRange.schema.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
@kriswest Looks like tooling generated the newly proper cased timeRange side by side with the existing lower timerange.schema.json.

Reply:
Apologies @bingenito I don't quite understand the comment. 

- **In 2.1 and next** I'm only seeing the camelCased `fdc3.timeRange` schema and generated code in JS and markdown and NOT the uncased `fdc3.timerange`.
- **In 2.0 (only)** I see the uncased `fdc3.timerange`

I don't see both in one version - although I am amazed at the number of issues one mistake on the casing of a character (my bad) has caused!

Reply:
@kriswest 
In next it is already present as lower case - https://github.com/finos/FDC3/blob/main/website/static/schemas/next/context/timerange.schema.json

This PR has generated and is now adding the camelCase side by side as it was somehow never there.

Reply:
Gah, of course that doesn't happen to me locally as the camelcased one is on my filesystem and ensures the case is retained. Behaviour will be different on a case-sensitive file system. Note that the content of that file is actually the camel-cased version!

I dealt with that in the source files, but clearly not in the website's static folder.

Heres a PR that fixes that (I hope) @bingenito. @sivaprasadanakarla you can ignore this as it should merge into your PR fine.

Reply:
Actually, @sivaprasadanakarla you must be on a case-insensitive filesystem. I'll get back to you once we have this fix merged as you may need to resolve a conflict once it's in. Nothing to do right now however.

Reply:
Merged. @sivaprasadanakarla please merge changes from the main branch into your PR branch and then if 
[/website/static/schemas/next/context/**timerange**.schema.json](https://github.com/sivaprasadanakarla/FDC3/blob/Issue%231371/website/static/schemas/next/context/timerange.schema.json) still exists, delete it and make sure you *only* have [/website/static/schemas/next/context/**timeRange**.schema.json](https://github.com/sivaprasadanakarla/FDC3/blob/Issue%231371/website/static/schemas/next/context/timeRange.schema.json)

Reply:
@kriswest working on it.